### PR TITLE
Refactor env settings

### PR DESCRIPTION
**Pull request by serge1peshcoff**
Fixes MEMB-676, required for moving to @aegee.eu mailing.

Changes:
1) removed secrets, moved everything to .env file
2) that's pretty much it :D

### CODE REVIEW

**Comment by serge1peshcoff** [comment]
@linuxbandit ping

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -31,9 +32,11 @@ services:
         image: aegee/oms-mail-transfer-agent:latest
         expose:
             - "25"
-        secrets:
-            - mail_password
-            - mail_user
+        environment:
+            SMTP_HOST: ${SMTP_HOST}
+            SMTP_PORT: 587
+            SMTP_USER: ${SMTP_USER}
```
Comment (linuxbandit) [inline docker/docker-compose.yml]: Perhaps you could make the variables mandatory like you showed on slack

Comment (serge1peshcoff) [reply docker/docker-compose.yml]: done

Comment (serge1peshcoff) [reply docker/docker-compose.yml]: fuck, the tests are broken now. need to fix travis/circle ci, then it's done

**Pull request merged**